server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/materials/perforce_material_representer.rb

```diff
@@ -18,12 +18,16 @@ module ApiV1
   module Config
     module Materials
       class PerforceMaterialRepresenter < ScmMaterialRepresenter
+        P4MaterialConfig.send(:field_accessor, 'password')
```

Comment:
@ketan - the alternative would be introduce a new method in the java class which sets the password field directly without touching encryptedPassword, not sure what to call it though.